tests/redirects/map_globalconf.py

```diff
@@ -564,12 +570,18 @@
         url_test("/projects/security/older-vulnerabilities.html", "/security/known-vulnerabilities/older-vulnerabilities/"),
         # bug 1017564
         url_test("/mobile/RANDOM-STUFF/system-requirements/", "https://support.mozilla.org/kb/will-firefox-work-my-mobile-device"),
-        # bug 1041712, 1069335, 1069902
-        url_test(
-            "/{firefox,mobile}/{2,19,27}.0{a2,beta,.2}/{release,aurora}notes/{,stuff}",
-            "http://website-archive.mozilla.org/www.mozilla.org/firefox_releasenotes/en-US"
-            "/{firefox,mobile}/{2,19,27}.0{a2,beta,.2}/{release,aurora}notes/{,stuff}",
-        ),
+        # bug 1041712, 1069335, 1069902; superceded by Issue 16381
+        # url_test(
+        #     "/mobile/{2,19,27}.0{a2,beta,.2}/{release,aurora}notes/{,stuff}",
+        #     "http://website-archive.mozilla.org/www.mozilla.org/firefox_releasenotes/en-US"
```

Comment:
wondering whether the links to archive from WMO wouldn't be better straight on the first hop from bedrock, instead of passing that through FXC…

Reply:
Or is it already too complex to weed through the rules and pinpoint the ones that should redirect selectively elsewhere right away?

Reply:
There's so much complexity in here that I think it would take a lot longer to ship, so having to keep the changes lighter for now